admin_guide/cluster_management.rst

```diff
@@ -668,9 +668,7 @@ In the new db on every worker, manually run:
 
 Now the new database will be operating as another Citus cluster.
 
-.. _phone_home:
-
-Checks For Updates and Cluster Statistics
+Checks For Updates
 =========================================
 
 Unless you opt out, Citus checks if there is a newer version of itself during installation and every twenty-four hours thereafter. If a new version is available, Citus emits a notice to the database logs:
```

Comment:
How does one opt out of version checks? Is it `citus.enable_statistics_collection = off`? Do we no longer do update checks?

Reply:
In https://github.com/citusdata/citus/blob/master/src/backend/distributed/utils/maintenanced.c#L339 it looks as though `CheckForUpdates()` is called only when `EnableStatisticsCollection` (aka `citus.enable_statistics_collection`) is true.

Reply:
It turns out CheckForUpdates doesn't really work, since we never update the advertised version: https://reports.citusdata.com/v1/releases/latest

I'd propose removing this from docs